tests/dags/test_dummy_debug_dag.py

```diff
@@ -13,16 +13,16 @@ def test_dummy_debug_dag_has_two_tasks_and_order(airflow_dag_bag):
     assert dag is not None
     tasks = dag.tasks
     task_ids = list(map(lambda task: task.task_id, tasks))
-    assert {'feature_engineering', 'model_training'}.issubset(set(task_ids))
+    assert {'check_step_1', 'check_step_2'}.issubset(set(task_ids))
```

Comment:
we could in principle write automation of structure checking tests given a arbitrary Pipeline